practices/javascript-styling.md

```diff
@@ -0,0 +1,35 @@
+# Javascript dependent styling
+
+A class name of `.js` on the document root element (normally `html`) will be used to bootstrap JS-only visual styles. We do this to prevent the visual flicker and browser reflow that may occur when JavaScript applies CSS styles to a component after the page has loaded.
```

Comment:
```suggestion
A class name of `.js` on the document root element (normally `html`) is used to target JS-only visual styles via CSS. We do this to minimise any visual flickering and browser reflows that may occur when JavaScript applies CSS styles to a component after the page has loaded.
```

I _think_ my suggestion is a little more accurate. But I am not clear on what the second sentence is saying -- it doesn't I think prevent reflows etc? Or is this in comparison to another, unmentioned technique?

Reply:
Maybe "JS-dependent" rather than "JS-only".

Reply:
nice, done ty